ovos_plugin_common_play/ocp/__init__.py

```diff
@@ -125,10 +127,15 @@ def handle_home(self, message=None):
 
     @property
     def using_new_pipeline(self) -> bool:
-        # TODO - default to True in ovos-core 0.1.0
-        # more info: https://github.com/OpenVoiceOS/ovos-core/pull/456
-        moved_to_pipelines = Configuration().get("intents", {}).get("experimental_ocp_pipeline")
-        return moved_to_pipelines
+        # this is no longer configurable, most of this repo is dead code
+        # keep this check to allow smooth updates from the couple alpha versions this was live
+        if Configuration().get("intents", {}).get("experimental_ocp_pipeline"):
+            return True
+        # check for min version for default ovos-config to contain OCP pipeline
+        from ovos_config.version import VERSION_BUILD, VERSION_ALPHA, VERSION_MAJOR, VERSION_MINOR
+        if VERSION_BUILD > 13 or VERSION_MAJOR >= 1 or VERSION_MINOR >= 1:
+            return True
+        return VERSION_BUILD == 13 and VERSION_ALPHA >= 14
```

Comment:
for anyone reviewing this is the relevant change to account for https://github.com/OpenVoiceOS/ovos-core/pull/491 and https://github.com/OpenVoiceOS/ovos-config/pull/96